src/display/Display.spec.js

```diff
@@ -1 +1,38 @@
 // Test away!
+import React from 'react';
+import Display from './Display';
+import {render, cleanup} from 'react-testing-library';
+
+afterEach(cleanup);
+
+describe('Display renders without crashing.', () => {
```

Comment:
I like your testing so far. It is very organized and consistent. I do have a question though. Do you believe all the "it" statements should be under this "Display renders without crashing."? I feel that this may be too general. You should consider adding another describe where the logic of the program is tested. I would also expand the display renders tests with more tests if you feel up to it.